@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -535,215 +512,7 @@ export type BaseStyle<Technique, Params> = Partial<Params> & {
     attr?: Partial<Params>;
 };
 
-/**
- *
- * @defaultSnippets [
```

Comment:
Did this work before? Would be a nice feature to have theses snippets but I guess they could be in a separate file.